balancer/frontend-v2
# Update featured protocols

# Description

Please include a summary of the change and if relevant which issue is fixed. Please also include relevant motivation and context.

## Type of change


## How should this be tested?

Check to make sure it looks like this:

![featured-protocols](https://user-images.githubusercontent.com/1121139/227265118-1a21b2f4-35d9-4739-b8df-b4e1acd291f8.png)

## Checklist:
